lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -381,6 +393,88 @@ def stop_event_loop(self):
     stop_event_loop.__doc__=FigureCanvasBase.stop_event_loop_default.__doc__
 
 
+_flow = [Gtk.Orientation.HORIZONTAL, Gtk.Orientation.VERTICAL]
+flow_types = ['horizontal', 'vertical']
+
+
+class Window(WindowBase, Gtk.Window):
```

Comment:
:+1: On making the GUI specific classes have the framework name in them and doing `Window = WindowGTK3` at the end.  It helps to keep the code a bit more organized.  I have poked around in some projects that aggressively shadow names between modules and it gets bad.